Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -191,6 +206,37 @@ static bool HasPlugin(Assembly assembly)
             }
         }
 
+        private static bool IsExtensionInAssemblyModelType(string assemblyMetadataPath)
+        {
+            var assemblyMetadata = Load(assemblyMetadataPath);
+
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(TemplateSetConventions.Extension);
+        }
+
+        private static bool IsProfileInAssemblyModelType(string assemblyMetadataPath)
+        {
+            var assemblyMetadata = Load(assemblyMetadataPath);
+
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(TemplateSetConventions.Profile);
+        }
+
+        public static AssemblyMetadata Load(string fileFullName)
+       {
+           if (!File.Exists(fileFullName))
+           {
+               throw new FileNotFoundException($"Non-existent file path provided. Expected location {fileFullName}.");

```

Comment:
Wrap all message parameters in single quotes (i.e. add quotes around `{fileFullName}').

Suggest changing the message to `$"Assembly metadata file '{fileFullName}' not found."`

Reply:
yes ,  I updated . Thanks